### PR TITLE
132 reactive search routes when token expire logout

### PR DESCRIPTION
**Pull request by longshuicy**
To test:
Set the token expiration in keycloak admin console (http://localhost:8080/keycloak) to 1 minute; and type in the search box see what happened after 1 minute
image

### CODE REVIEW

File: frontend/src/components/search/SearchErrorBoundary.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	return (
+		<ErrorBoundary
+			renderError={error => (
+				<>
+					{
+						(() => {
+							if (error["status"] === 401 || error["status"] === 403) {
+								V2.OpenAPI.TOKEN = undefined;
+								cookies.remove("Authorization", {path: "/"});
+								return <Navigate to="/auth/login"/>;
```
Comment (longshuicy) [inline frontend/src/components/search/SearchErrorBoundary.tsx:24]: I cannot properly do the refresh token logic here; for now just redirect to the login page

File: frontend/src/components/search/SearchErrorBoundary.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+							} else {
+								// TODO add prettier message or report function
+								return <h1>An error has happened.</h1>
```
Comment (longshuicy) [inline frontend/src/components/search/SearchErrorBoundary.tsx:27]: This will trigger when elasticsearch is dead (e.g. 500 internal error)

**Comment by lmarini** [comment]
There is a build error of the frontend when installing dependencies.

```
#12 3.046 npm ERR! 
#12 3.048 npm ERR! Could not resolve dependency:
#12 3.050 npm ERR! peer webpack@"^5.0.0" from css-minimizer-webpack-plugin@2.0.0
#12 3.051 npm ERR! node_modules/css-minimizer-webpack-plugin
#12 3.053 npm ERR!   dev css-minimizer-webpack-plugin@"^2.0.0" from the root project
```

**Comment by longshuicy** [comment]
> There is a build error of the frontend when installing dependencies.
> 
> ```
> #12 3.046 npm ERR! 
> #12 3.048 npm ERR! Could not resolve dependency:
> #12 3.050 npm ERR! peer webpack@"^5.0.0" from css-minimizer-webpack-plugin@2.0.0
> #12 3.051 npm ERR! node_modules/css-minimizer-webpack-plugin
> #12 3.053 npm ERR!   dev css-minimizer-webpack-plugin@"^2.0.0" from the root project
> ```

I merge main (where it has the webpack 5 and other dependency change) into the branch that this branch is merged into
See if the check pasts here: https://github.com/clowder-framework/clowder2/pull/134

**Comment by longshuicy** [comment]
> Can you resolve conflicts? Thank you!

done

**Review by lmarini — APPROVED** [review]

**Pull request merged**